vendor/github.com/openshift/installer/data/data/bootstrap/systemd/units/fluentbit.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
+After=network-online.target
+
+[Service]
+EnvironmentFile=/etc/sysconfig/fluentbit
+ExecStartPre=-/bin/podman rm -f fluent-journal
+ExecStartPre=-/bin/podman pull $FLUENTIMAGE
+ExecStartPre=-mkdir -p /var/lib/fluent
+ExecStart=/bin/podman run \
+  --entrypoint /opt/td-agent-bit/bin/td-agent-bit \
+  --net=host \
+  --hostname `hostname` \
+  --name fluent-journal \
+  --rm \
+  -v /etc/fluentbit/journal.conf:/etc/fluentbit/journal.conf \
+  -v /var/lib/fluent:/var/lib/fluent:z \
+  -v /var/log/journal:/var/log/journal:z,ro \
+  -v /etc/machine-id:/etc/machine-id:ro \
+  $FLUENTIMAGE \
+  -c /etc/fluentbit/journal.conf
+
+ExecStop=/bin/podman stop %N
+Restart=always
```

Comment:
check how we use [Unit] StartLimitInterval=0 and [Service] RestartSec=1 elsewhere.  On RHCOS it might need to be StartLimitIntervalSec.  Might be worth pulling those in to these service files too.

Reply:
Removed the rate limiting on starts/restarts. Set restartsec to 1s